HISTORY.rst

```diff
@@ -3,6 +3,62 @@
 Release History
 ---------------
 
+2.0.0 (2023-04-15) Wellton
+++++++++++++++++++++++++++
+
+**Special notes**
+
+Mesa 2.0 includes several breaking changes that provide significant improvements to Mesa.
+
+**Breaking Changes:**
+
+* space: change `coord_iter` to return `(content,(x,y))` instead of `(content, x,y)`; this reduces known errors of scheduler to grid mismatch #1566, #1723
+* space: change NetworkGrid `get_neighbors` to `get_neighborhood`; improves performance #1542
+* space: raise exception when pos is out of bounds in `Grid.get_neighborhood` #1524
+* space: remove deprecations (#1520, #1687, #1688):
+    * `find_empty()`: convert this to `move_to_empty()`
+    * `num_agents`: removed parameter from `move_to_empty()`
+    * `position_agent()`: convert this to `place_agent`
+    * `neighbor_iter()`: convert this to `iter_neighborhood()`
+* batchrunner: remove deprecations #1627
+    * `class BatchRunner` and `class BatchRunnerMP`: convert these to `batch_run()`
+* visualization: easier visualization creation #1693
+    * `UserSettableParameter(['number', 'slider','checkbox', 'choice', `StaticText`]): convert to `NumberInput`,`Slider`, `CheckBox`, `Choice`, `StaticText`
```

Comment:
Can you add an example by linking to https://github.com/projectmesa/mesa-examples/blob/db2ec0383eb3b1868e91c828101e84cce97bbb63/examples/boltzmann_wealth_model/boltzmann_wealth_model/server.py#L25-L32. I want to make sure people don't have problem migrating.